src/krkn_lib/k8s/pod_monitor/pod_monitor.py

```diff
@@ -0,0 +1,286 @@
+import os
+import re
+from concurrent.futures import Future
+from concurrent.futures.thread import ThreadPoolExecutor
+from functools import partial
+
+from kubernetes import client, config, watch
+from kubernetes.client import V1Pod
+
+from krkn_lib.models.pod_monitor.models import (
+    PodsSnapshot,
+    MonitoredPod,
+    PodEvent,
+    PodStatus,
+)
+
+config.load_kube_config(os.path.join(os.environ["HOME"], ".kube/config"))
```

Comment:
do we need this here? can we use the loaded kubeconfig from k8s/krkn_kubernetes?

Reply:
typos from the POC thanks for spotting!